aquacomputer_d5next.c

```diff
@@ -64,6 +64,7 @@ static u8 secondary_ctrl_report[] = {
 #define AQC_FAN_CURRENT_OFFSET		0x04
 #define AQC_FAN_POWER_OFFSET		0x06
 #define AQC_FAN_SPEED_OFFSET		0x08
+#define AQC_FAN_PWM_OFFSET			0x01
```

Comment:
we might want to separate this into it's own block and name it `AQC_FAN_CTRL_PWM_OFFSET`, since it's an offset into a different block